grobid-core/src/main/java/org/grobid/core/document/DocumentNode.java

```diff
@@ -182,5 +184,12 @@ else if (father == null) {
              }
          }
      }*/
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
```

Comment:
not used?

Reply:
it's used at line 94 of the `PDFALTOOutlineSaxHandler.java `